name: mobile_robot
T: 0.1
N: 20
l_r: 1.3
l_f: 1.3
wheelbase: 2.60

constraints:
  v_max: 20
  v_min: -20
  v_ref: 5

  delta_max: 0.42
  delta_min: -0.42
  x_min: -1000.0
  x_max: 1000.0
  y_min: -1000.0
  y_max: 1000.0

costs:
  y_cost: 2
  x_cost: 2
  yaw_cost: 0.5
  v_cost: 1 
  steer_cost: 0.
  delta_v_cost: 0.25
  delta_steer_cost: 0.5